lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -31,7 +31,7 @@
 options:
     backup:
         description:
-            - Path to store a backup of the initial certificate, if I(path) pointed to an existing file certificate.
+            - The destination path for a backup of the certificate in I(path).
```

Comment:
```suggestion
            - Whether a backup should be made for the certificate in I(path).
```
After all, it's a boolean option, and not the return value (which will contain the path) :)